### ISSUE TITLE
Switching between table view and line view changes y-axis metric

### ISSUE THREAD

**Issue by yugandharb** [yahoo/navi#820]
**Describe the bug**
Please find attached screenshots of the issue that I see (unfortunately I cannot post the entire chart due to sensitive information). The default view is line chart (report saved), when I switch to the table view and navigate back to the line chart view the metric on the y-axis changes automatically.

**To Reproduce**

**Expected behavior**

**Screenshots**
![Screen Shot 2020-05-26 at 12 03 27 PM](https://user-images.githubusercontent.com/6739010/82929421-3b078900-9f49-11ea-8efd-a634a99fd3fe.png)
![Screen Shot 2020-05-26 at 12 03 45 PM](https://user-images.githubusercontent.com/6739010/82929431-3f33a680-9f49-11ea-825b-50f43d851222.png)

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser Chrome
 - Version 81.0.4044.138 (Official Build) (64-bit)

**Additional context**

**Comment by cythrawll** [comment]
This is a known drawback. When you switch visualizations the settings of the new visualization settings overwrite the new one, so when you switch back... you have to reconfigure.  The main use case is someone switching to the table to look at the raw data. We have some ideas on how to tackle that by giving a data viewer that isn't a visualization.  We also have plans to persist more series/style configuration even when the visualization is not compatible.

**Comment by yugandharb** [comment]
@cythrawll thanks, I see a similar issue where the sub totals on the data table disappear/do not persist. Is that a known issue as well? if not, I can open an another issue

**Comment by cythrawll** [comment]
Yes same issue, every time you switch visualizations, the vis settings go back to default.
